backend/schema/heat_pump_water_heater.ts

```diff
@@ -6,8 +6,14 @@ export const heatPumpWaterHeaterProperties = {
   tankCapacityGallons: {
     type: "number",
   },
+  uef: {
+    type: "number",
+  },
+  fhr: {
+    type: "number",
+  },
```

Comment:
I think we should use full names – uniformEnergyFactor and firstHourRating.

If the frontend wants to use abbreviations, that's fine, but the canonical backend name should be more interpretable.

Reply:
Agreed!